tests/system/action/motion/test_create_forwarded.py

```diff
@@ -50,19 +54,94 @@ def test_correct_origin_id_set(self) -> None:
                 "meeting_id": 2,
                 "origin_id": 12,
                 "text": "test",
+                "reason": "reason_jLvcgAMx",
             },
         )
         self.assert_status_code(response, 200)
+        model = self.assert_model_exists(
```

Comment:
Please test the correct submitter in target-motion

Reply:
Add code to test for submitter, motion_submitter and link in user/2.